api/app/GraphQL/Mutations/ClosePool.php

```diff
@@ -15,7 +15,10 @@ final class ClosePool
     public function __invoke($_, array $args)
     {
         $pool = Pool::find($args['id']);
-        $pool->update(['closing_date' => Carbon::now()]);
+        $pool->update([
+            'closing_date' => Carbon::now(),
+            'closing_reason' => isset($args['closing_reason']) ? $args['closing_reason'] : null,
+        ]);
```

Comment:
Should the `ChangePoolClosingDate` mutation then null out `closing_reason`

Reply:
That makes sense to me, updated in [merge change closing date validators](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9945/commits/6a45f80cb07993c28b511cdcb2cbb6dc8ff00835)